packages/sources/src/addSourceWizard/hardcodedSchemas.js

```diff
@@ -374,13 +374,21 @@ export default {
                             initializeOnMount: true
                         }]
                     }, {
-                        title: <FormattedMessage id="cost.arn.tagsStepTitle" defaultMessage="Activate cost allocation tags" />,
+                        title: <FormattedMessage id="cost.arn.tagsStepTitle" defaultMessage="Tags, Aliases and Org Units" />,
```

Comment:
Since it is wrapping to a second line already, we should just go ahead and write out the full word "Org" to be "Organizational" for the step's title.  So use "Tags, aliases, and organizational units" as the step's title.